### ISSUE TITLE
parse error

### ISSUE THREAD

**Issue by jonmorehouse** [benmorehouse/rtc#3]
Hi @benmorehouse I downloaded this today (was playing around with the tool generally) and ran into the following problem:

```bash
2020/04/23 11:54:50 Database err: Bucket not found ---> suggestion: run ./rtc init
~clones/rtc (master)$ rtc init
Welcome to your very own Rescue Time CLI
Have you created a rescuetime account yet? [Y/N]y
Next go to https://www.rescuetime.com/apidoc and create an API key.
This will act as your way of getting access to your account through this CLI.
Have you created a rescuetime API Key yet? [Y/N]y
Paste your API Key here:xxxxx
~clones/rtc (master)$ rtc
2020/04/23 11:55:49 parse error on line 1, column 1: bare " in non-quoted-field
~clones/rtc (master)$
```
(Redacted api key).

Any suggestions? I'm running this on `go1.14` per:

```bash
$ ~clones/rtc (master)$ go version
go version go1.14 darwin/amd64
```

**Comment by benmorehouse** [comment]
My initial thoughts:
- this application should come w/ it's own configuration file of some sort. reflecting on the project, the current schema of how I hold the api key is inefficient. My proposal is that instead I am going to create a configuration file for the project to hold the api key for the user. This will more so fix the extra step that the user has to do, and could cause bottlenecks down the line. 
- I was not able to recreate the issue you had however i think that the first point can help combat it. More to come on that before closing this issue.

@jonmorehouse

**Comment by benmorehouse** [comment]
@jonmorehouse 
I gave the issue a further look based on your feedback and have taken action on the following:

- before, the tool was using a remote database to keep track of the API key. At the time, it was designed to be a possible way to hold onto your data for when you are offline. This turned into a design flaw, as it was quite easy to trigger a bottleneck in the way we hold the api key for rescuetime. I have refactored the process to ditch the embedded database and instead use a dotfile (formally called .rtc.yaml) which will hold onto the api key, amongst other things.

- I also became bothered by the fact that the first time you used rtc, it showed you an ugly message like what you showed me and made you type in another command. Now upon using the CLI for the first time, it should use default values and prompt the user right away to get the API key. 

Will be releasing an update to v 1.0 shortly. Thanks for the feedback!
